docs/docs/marketplace/plugins/github.md

```diff
@@ -0,0 +1,101 @@
+---
+id: marketplace-plugin-github
+title: GitHub
+---
+
+ToolJet can connect to GitHub account to read and write data. In order for ToolJet to access and manipulate data on GitHub, a **GitHub Personal Access Toke**n is necessary to authenticate and interact with the GitHub API.
+
+<div style={{textAlign: 'center'}}>
+
+<img className="screenshot-full" src="/img/marketplace/plugins/github/githubadd.gif" alt="Marketplace: GitHub" />
+
+</div>
+
+:::note
+Before following this guide, it is assumed that you have already completed the process of **[Using Marketplace plugins](/docs/marketplace/marketplace-overview#using-marketplace-plugins)**.
+:::
+
+## Connection
+
+For connecting to GitHub, following credentials are required:
```

Comment:
Would it be possible to include a note stating that if a personal token is not provided, data from private repositories will not be retrieved via the GitHub Plugin? It should be noted, however, that public repository data can still be retrieved.